metadata_backend/api/operators.py

```diff
@@ -214,14 +219,20 @@ async def _update_object_from_db(self, schema_type: str,
                           "was not found.")
                 LOG.error(reason)
                 raise web.HTTPNotFound(reason=reason)
-            replace_success = (await self.db_service.update(schema_type,
-                                                            accession_id,
-                                                            data))
+            update_success = (await self.db_service.update(schema_type,
```

Comment:
regarding this: "The reason to keep it like this is that PATCH is only done on draft and folders, not actual objects and we should not do PATCH on objects, that will open up for other issues (e.g. accidental overwrite of objects with invalid data)."

If we don't want to do patches for objects, shouldn't we just prevent it in handlers.py in the first place
https://github.com/CSCfi/metadata-submitter/blob/6afe4c4a866fbcbaa8d916f92678c4dfc429e688/metadata_backend/api/handlers.py#L296

Reply:
it is not in `server.py` that is enough to say Method not allowed

Reply:
but I can add a check there just to be extra, extra sure.